html/view.php

```diff
@@ -2,9 +2,11 @@
 require_once('db.php');
 require_once('functions.php');
 
-/* 返信課題はここからのコードを修正しましょう。 */
 
-/* 返信課題はここまでのコードを修正しましょう。 */
+require_once('db.php');
+$t = getTweet($_GET['id']);
+$t=$t[0];
```

Comment:
同じ変数 `$t` に再代入するのは可読性が低下してよくないので、こうしましょう
```suggestion
$t = getTweet($_GET['id'])[0];
```